extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -161,8 +161,22 @@ export class AmpAnalytics extends AMP.BaseElement {
           if (!result) {
             return;
           }
-          addListener(this.getWin(), trigger,
-              this.handleEvent_.bind(this, trigger));
+
+          if (trigger['selector']) {
+            // Expand the selector using variable expansion.
+            trigger['selector'] = this.expandTemplate_(trigger['selector'],
+                trigger);
+            return urlReplacementsFor(this.getWin()).expand(trigger['selector'])
```

Comment:
One concern is that since uppercase CSS are allowed and the var names are pretty logical - we have a meaningful possibility of an accidental collision. WDYT? Is url replacements part necessary here?